File: Logica_de_Programacao/Desafio_9/desafio_9/Desafio_9.py
numero1 = float(input("Digite o primeiro número: "))

while True:
    numero2 = float(input("Digite o segundo número (não pode ser zero): "))

    if numero2 != 0:
        break
    else:
        print("O divisor não pode ser zero! Digite um valor válido.\n")

resultado = numero1 / numero2
print(f"\nO resultado da divisão de {numero1} por {numero2} é: {resultado:.2f}")

File: Logica_De_Programacao_Java/Desafio_9/src/Desafio_9.java
import java.util.Scanner;

public class Desafio_9 {
    public static void main(String[] args) {

        Scanner entrada = new Scanner(System.in);

        double valor1, valor2;
        double resultado;

        System.out.print("Digite o primeiro valor: ");
        valor1 = entrada.nextDouble();

        do {
            System.out.print("Digite o segundo valor: ");
            valor2 = entrada.nextDouble();

            if (valor2 == 0) {
                System.out.println("Valor inválido. Digite um valor diferente de zero.");
            }
        } while (valor2 == 0);

        resultado = valor1 / valor2;

        System.out.println("O resultado da divisão é: " + resultado);

        entrada.close();
    }
}